src/Matterhorn/Command.hs

```diff
@@ -276,6 +276,13 @@ commandList =
 
   , Cmd "move-team-right" "Move the currently-selected team to the right in the team list" NoArg $ \_ ->
         moveCurrentTeamRight
+  
+  --, Cmd "favorite-channel" "Mark the current channel as favorite/non-favorite" NoArg $ \_ -> do
```

Comment:
Similarly to my comment on this part of the code in a different patch, please leave comment hunks like this out of your commits. I know it can be helpful to keep those around for your own development, but they don't need to be committed (and can lead to confusion).

Reply:
Noted.